programs/streamflow-sdk/src/lib.rs

```diff
@@ -203,7 +338,15 @@ pub mod streamflow_sdk {
         can_update_rate: bool,
     ) -> Result<()> { Ok(()) }
 
-    /// Anchor rpc handler used for CPI code generation
+    /// Update a Stream
+    ///
+    /// This method allows to enable automatic withdrawals and/or update `amount_per_period` in a Stream
```

Comment:
Phrasing is a bit strange to me. I'd go with: This method enables automatic .. and/or ...